.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,145 @@
+        
+        </details>
+    validations:
+      required: false
+
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional context
+      description: Add any other context about the problem here (e.g., relevant .env settings without sensitive data).
+    validations:
+      required: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)